iterationTask.js

```diff
@@ -0,0 +1,63 @@
+// 1
+const products = {
+  0: 'Phone',
+  1: 'Tracker',
+  2: 'Tv',
+  [Symbol.iterator]: function () {
+      let count = 0;
+      
+      return {
+          next() {
+              return {
+                  value: products[count++],
+                  done: count > Object.keys(products).length
+              };
+          }
+      };
+  }
+};
+
+// 2
+const obj = {
+  zone: true,
+  word: 'test',
+  skill: 'JS',
+  name: 'John',
+  [Symbol.iterator]: function () {
+    let count = 0;
+    
+    return {
+        next() {
+            return {
+                value: obj[Object.keys(obj).sort()[count++]],
+                done: count > Object.keys(obj).length
```

Comment:
you shouldn't calculate keys length and make an array sorting on every iteration